content/post/mv-reflection-002.md

```diff
@@ -0,0 +1,44 @@
+>
+> **Livingston:** That was so generous.
+>
+> **Wozniak:** But it’s that whole thing I was talking about: Hewlett-Packard, we’re a community. There was a recession in ’73 and Hewlett-Packard had to cut back 10 percent. Instead of laying off 10 percent of the people, they cut everyone’s salary by 10 percent and gave us one day off every two weeks. So basically they said “nobody goes without a job.” And I like that sort of thing. So a bunch of Apple engineers and marketing people got to benefit from going public. Otherwise, they’d have no stock at all. Mike Markkula kind of felt that some of these people didn’t deserve it; some people shouldn't get stock. But I disagreed with him on that. Nobody stopped me, so I did it.
+
+What struck me here is the inspiration for his generosity is not his vietnam-era politics, not some communitarian startup ethos, but the Hewlett-Packard corporate culture. And as a counterpoint, Apple the scrappy startup is already screwing over early employees 4 years into its existence.
+
+I'm curious about what made Hewlett-Packard so great in Woz's eyes and why. He probably didn't own much stock, it certainly wasn't a worker-owned co-op. What aspect of the culture made him feel so fulfilled and so loyal?
+
+I want to ask other than the practicalities of corporate structures that make people feel ownership, what aspects of workplace culture make people feel fulfilled? I ask this question holding the truth that culture stuff can be used as a distraction while the real material gains are taken away - you don't get to keep any equity but boy is the culture great! Holding that critical lens I think the next thing I should read should be about fulfilling workplace cultures.
```

Comment:
I guess building on my comment above, *trust* seems huge, and by extension, relationships. But also from that comment, scaling that seems challenging. "How do we scale this marriage?" lol...